config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
-  root to: 'organisations#index'
+  root :to => 'organisations#index'
```

Comment:
Ahh this must have creeped in when I rebased against previous changes.. will fix it.